client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -254,6 +255,10 @@ export default {
       return today
     },
 
+    filterMenue () {
```

Comment:
sorry, that I didn't see it before:
This is a method, but its more like an dynamic variable, so the wording should be `filteredMenuEntries`

Welcome to the world of vue ;-)

Reply:
like alice in wonderland :)) https://github.com/demos-europe/demosplan-core/pull/4130/commits/b207b6293b2842890d0625246234926e7ee9a5e2